Extension/package.json

```diff
@@ -4034,17 +4034,20 @@
       "[cpp]": {
         "editor.wordBasedSuggestions": false,
         "editor.suggest.insertMode": "replace",
-        "editor.semanticHighlighting.enabled": true
+        "editor.semanticHighlighting.enabled": true,
+        "editor.inlayHints.fontSize": 11
```

Comment:
Is this guaranteed to give us what we want?  What if the normal font size is already 11?

They should have done it like the code lens setting:
![image](https://user-images.githubusercontent.com/12818240/174690383-d69f67ac-4c46-42c9-a2fe-98499a79d4b0.png)

Reply:
I sent a message to isidorn, but I think it would be good to open an issue as well.  If you do that, I'll forward the issue to him.

Reply:
It's ok to ship this for now, I think, since we can undo it later.

Reply:
This is for 1.10.1 and not 1.10.0 so we can get feedback both ways.

Also, the default editor.fontSize is 14, so VS is rendering the inlay hint at 80% of the editor font (my original screenshot comparison was comparing it with the incorrect default of 12). 

I've filed a feature request on VS at https://github.com/microsoft/vscode/issues/152690

Reply:
And FYI the difference in pixel height for the "c" of "const" is 8/9/10 pixels for 11/12/14 font sizes.

Reply:
FYI, I changed it from 11 to 12 -- I've updated the description and screenshots to explain why.